app/assets/stylesheets/app/repository.scss

```diff
@@ -0,0 +1,8 @@
+.repository {
+  border-top: solid 1px $light-border;
+  height: 73px;
```

Comment:
Properties should be ordered border-top, display, height, justify-content, padding-left, padding-top